filter_lists/regional.json

```diff
@@ -212,7 +212,7 @@
     {
         "uuid": "1C6D8556-3400-4358-B9AD-72689D7B2C46",
         "url": "https://raw.githubusercontent.com/finnish-easylist-addition/finnish-easylist-addition/master/Finland_adb.txt",
-        "title": "Finnish Addition to Easylist",
+        "title": "Easylist Finnish Addition Filterlist",
```

Comment:
@antonok-edm — should this be "Easylist, Finnish Addition to Filterlist" instead? Seems like we need "to" and a comma in there. Or maybe "Finnish Easylist Addition to Filterlist" could work, too.

Reply:
Simplified it to 'Easylist Finnish'